scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -323,15 +323,23 @@ def update_policy_gate_url(json_data, pl_key, execution_str):
         raise e
 
 
-def get_sql_pi_executions(sqlcursor_orca):
+def get_sql_pi_executions_mysql(sqlcursor_orca):
     try:        
-        sqlcursor_orca.execute("SELECT pi.application, pi.body, ps.body FROM pipeline_stages ps LEFT OUTER JOIN pipelines pi ON ps.execution_id = pi.id")
+        sqlcursor_orca.execute("SELECT pi.application, pi.body, ps.body FROM pipeline_stages ps LEFT OUTER JOIN pipelines pi ON ps.execution_id = pi.id where (JSON_EXTRACT(ps.body, '$.type') = 'verification') or (JSON_EXTRACT(ps.body, '$.type') = 'policy') or (JSON_EXTRACT(ps.body, '$.type') = 'approval')")
         return sqlcursor_orca.fetchall()        
     except Exception as e:
         print("Exception occurred while getting the pipeline executions : ", e)
         logging.error("Exception occurred while getting the pipeline execution : ", exc_info=True)
         raise e
 
+def get_sql_pi_executions_postgres(spindb_orca_postgres):
+    try:        
+        spindb_orca_postgres.execute("SELECT pi.application, pi.body, ps.body FROM pipeline_stages ps LEFT OUTER JOIN pipelines pi ON ps.execution_id = pi.id where (ps.body::jsonb ->> 'type' @@ 'verification') or (ps.body::jsonb ->> 'type' @@ 'policy') or (ps.body::jsonb ->> 'type' @@ 'approval')")
```

Comment:
there is one more type of custom gate. 'testVerification' . Please add that as well

Reply:
Updated as per suggested